cohort-engine/src/main/java/com/ibm/cohort/engine/measure/cache/TransientRetrieveCacheContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.cohort.engine.measure.cache;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+
+public class TransientRetrieveCacheContext implements RetrieveCacheContext {
```

Comment:
Transient in the  sense that cleanup does nothing?

Reply:
Transient in the sense that the data is never persisted outside of memory.  I could probably find a better word for that though.